metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractExecutablePlugin.java

```diff
@@ -171,6 +171,15 @@ DpsTask createDpsTaskForIndexPlugin(EcloudBasePluginParameters ecloudBasePluginP
     return createDpsTaskForProcessPlugin(ecloudBasePluginParameters, extraParameters);
   }
 
+  DpsTask createDpsTaskForDepublishPlugin(EcloudBasePluginParameters ecloudBasePluginParameters,
```

Comment:
The first parameter is never used. I guess you forgot to apply the base parameters?

Reply:
No actually they are not needed.. So I don't need the parameter here..